packages/redux/src/returns/selectors.ts

```diff
@@ -182,3 +182,20 @@ export const isReturnPickupCapabilityFetched = (
   (!!getReturnPickupCapability(state, returnId, pickupDay) ||
     !!getReturnPickupCapabilityError(state, returnId, pickupDay)) &&
   !isReturnPickupCapabilityLoading(state, returnId, pickupDay);
+
+export const getReturnsFromOrder = (
+  state: StoreState,
+  orderId: Order['id'],
+) => {
+  const returnEntities = getReturnsEntities(state);
+  if (!returnEntities) return;
```

Comment:
Prefer this form as it is easier to read and give an extra line of spacing before the if statement as well:

```suggestion
  if (!returnEntities) {
     return;
  }
```